src/pages/contact/index.tsx

```diff
@@ -17,25 +17,27 @@ const tags = CONTACT_DATA.map((contact) => {
   return contact.tags
 }).flat()
 const set = new Set(tags)
-const taglist = [...set]
+const tagFilter = [...set]
 
 const Contact = () => {
   const [filteredContacts, setFilteredContacts] =
     useState<Contact[]>(CONTACT_DATA)
 
-  const [selectedOption, setSelectedOption] = useState('')
+  const [selectedTag, setSelectedTag] = useState('')
```

Comment:
Might wanna add types to states (e.g. `useState<string>('')`)

Reply:
We pulled the lowercase naming convention from foodbank 💀

Reply:
Lowercase is fine for index files where the folder represents the component, but otherwise it's a bit 🤔  (docs recommend Pascal case: https://reactjs.org/docs/faq-structure.html).

Reply:
this should be fine for simple types as they can be inferred from the initial state, right?

Reply:
Was pretty simple to swap to regex, considers space, comma and string start a word start, being defensive including ',' hopefully not missing something. Changed file names and state type also.